src/ts/drawControl.ts

```diff
@@ -36,9 +36,10 @@ module powerbi.extensibility.visual {
                     if (feature.geometry.type === 'Point' && turf.booleanContains(sel_pol, feature)) {
                         return true;
                     }
-                    if ((feature.geometry.type === 'Polygon' || feature.geometry.type === 'Linestring') &&
-                       (turf.booleanOverlap(feature, sel_pol) || turf.booleanContains(sel_pol, feature) ||
-                        turf.booleanContains(feature, sel_pol)
+                    if (
+                        (feature.geometry.type === 'Polygon' || feature.geometry.type === 'LineString') &&
+                        (turf.lineIntersect(feature, sel_pol) || turf.booleanContains(sel_pol, feature) || 
```

Comment:
booleanOverlap can only compare same feature types, e.g. polygon vs polygon or lineString vs lineString, lineIntersect supports cross feature type compare so used this here instead